specifyweb/frontend/js_src/lib/localization/attachments.ts

```diff
@@ -226,4 +217,23 @@ export const attachmentsText = createDictionary({
   bulkImport: {
     'en-us': 'Bulk Import',
   },
+  onFile: {
+    'en-us': 'On File: {onFile: number} / {total: number}',
```

Comment:
should not have space after `:` (not sure if our localization library or weblate get's confused about that, but consistency is good - at least to not confuse the translators)

(optional) add a test that catches that